net/openvswitch/files/openvswitch.init

```diff
@@ -95,6 +95,29 @@ ovs_xx() {
 	esac
 }
 
+ovs_bridge_parse_datapath_id() {
+	local dpid dpif dpnet
+
+	case "$1" in
+		@*)
+			dpnet="${1##@}"
+			dpif="$(uci -q get network."${dpnet}".device)"
+			dpid="6f77$(cat /sys/class/net/${dpif}/address 2>/dev/null | sed 's/://g')"
+			;;
+		*)
+			dpif="$1"
```

Comment:
typo? `s/dpif/dpid/`

Reply:
Good catch. I made the error in my draft, then fixed it, then reworked everything again and made the same mistake. 🤦🏻